components/server/package.json

```diff
@@ -126,7 +126,7 @@
     "@types/js-yaml": "^4.0.3",
     "@types/lodash.debounce": "^4.0.6",
     "@types/mocha": "^10.0.1",
-    "@types/node": "^16.11.0",
+    "@types/node": "^18.18.8",
```

Comment:
why do we need 18 types for 16 node?

Reply:
We're using node 18 https://github.com/gitpod-io/gitpod/blob/main/components/server/leeway.Dockerfile#L5